src/sentry/static/sentry/app/types/hooks.ts

```diff
@@ -168,10 +168,7 @@ type FeatureDisabledHook = (opts: {
  */
 type AnalyticsInitUser = (user: User) => void;
 
-/**
- * Trigger analytics tracking in the hook store.
- */
-type AnalyticsTrackEvent = (opts: {
+export type AnalyticsTrackEventOptions = {
```

Comment:
I'd prefer we don't export this and instead use `Parameters<Hooks['analytics:track-event']>[0]`, we do this in other places

I can see the argument for wanting this exported as a named argument type though